meta-mender-core/classes/mender-setup-image.inc

```diff
@@ -1,4 +1,4 @@
-IMAGE_FSTYPES += "${@bb.utils.contains('MENDER_FEATURES', 'mender-image', ' mender mender.bmap', '', d)}"
+IMAGE_FSTYPES += "${@bb.utils.contains('MENDER_FEATURES', 'mender-image', ' mender', '', d)}"
```

Comment:
I don't understand the impact of removing it, best @kacf to review this :see_no_evil:

Reply:
Bmap is a filesystem-aware map file, which simply omits empty blocks from the file. But it has to be a raw filesystem file, it can't be a packed artifact.